devel/balena-cli/Portfile

```diff
@@ -0,0 +1,36 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+
+name                balena-cli
+version             12.55.2
+revision            0
+categories          devel
+platforms           darwin
+license             Apache-2
+maintainers         { ether.org.za:light @dylanbr } openmaintainer
+
+description         The official balena Command Line Interface.
+long_description    The balena CLI is a Command Line Interface for  \
+                    balenaCloud or openBalena. It builds on the balena API and \
+                    the balena SDK, and can also be directly imported in \
+                    Node.js applications.
+homepage            https://www.balena.io/
+distname            ${name}-v${version}-macOS-x64-standalone
+master_sites        https://github.com/balena-io/${name}/releases/download/v${version}
+
```

Comment:
Since the pre-built binary is an x86_64 binary, we need to add:
```
supported_archs x86_64
```

Reply:
Added.